service/storage_proxy.cc

```diff
@@ -493,7 +495,7 @@ class abstract_write_response_handler : public seastar::enable_shared_from_this<
     // return true if handler is no longer needed because
     // CL cannot be reached
     bool failure_response(gms::inet_address from, size_t count, error err) {
-        if (!_targets.contains(from)) {
+        if (boost::find(_targets, from) == _targets.end()) {
```

Comment:
There is also boost::algorithm::contains() that mirrors the pre-patch code more closely. However, I guess that after all the optimizations the resulting code will be the same.

Reply:
No, boost::algorithm::contains checks whether a range contains a subrange, not an element.

Reply:
Ok, you're right, I missed that. As if C++ wasn't confusing enough, Boost wants to help make it even more confusing ;-)